configure.ac

```diff
@@ -265,7 +265,7 @@ AC_MSG_CHECKING([if HDF5 version causes floating point exceptions with set flags
 AC_RUN_IFELSE([AC_LANG_PROGRAM([], [[
       use netcdf
       integer i, j
-      j = nf90_open("test.nc", NC_WRITE, i)
+      j = nf90_create("test.nc", NC_WRITE, i)
```

Comment:
Is it guaranteed that test.nc will not exist?  If so, we are fine.  If not, we should make it NC_OVERWRITE.

Reply:
I think autotools does the compile and runtime checks in some temp directory. Couldn't find anything specific in the docs but in my testing the check worked fine with that filename present in the build and source directories.